src/extension.ts

```diff
@@ -16,6 +16,10 @@ function transformEditorSelection(transformer: (text: string) => string) {
     });
 }
 
+function decodeURIComponentAndPlus(text: string): string {
+    return decodeURIComponent(text.replace('\+', ' '));
```

Comment:
Looks like this replaces only the 1st occurrence, I think you want to use a regex here

Reply:
Of course, you're right. 🤦‍♂️  Updated and added tests.